docker/README.md

```diff
@@ -146,3 +146,75 @@ Usage example agile-grasper:
+
+To start the command, enter the oneliner command, with ```-r true``` to pull the latest image.
+
+```bash <(curl -Ls http://bit.do/launch-sh) -r true```
+
+### Which image should I use: ```-i```
+
+As shown [above](#images) there are many different Docker images available for different tasks. In general, there are only a few that will be relevant for production use. The image that will be used is set using the ``` -i ``` flag when running the oneliner, as explained in the [previous section](#one-liner).
+
+#### Hand E
+
+The *shadowrobot/dexterous-hand* images contain the Hand E software. Unless there is a specific reason to use something else, the correct images to use are simply:
+* *shadowrobot/dexterous-hand:indigo*
+* *shadowrobot/dexterous-hand:kinetic*
+
+#### Hand H/Agile Grasper
+
+The *shadowrobot/agile-grasper* images contain the Hand H software. Hand H is only supported on ```kinetic```. In general, the correct image to use is the latest release version:
+* *shadowrobot/agile-grasper:kinetic-release*
+
+For this example, we'll start an *dexterous-hand:kinetic* docker:
+
+```bash <(curl -Ls http://bit.do/launch-sh) -r true -i shadowrobot/dexterous-hand:kinetic```
+
+##### Docker Hub user credentials: ```-u``` and ```-p```
+
+The Hand H software and Docker Images are private. This means that you'll need to provide the one-liner with credentials for Docker Hub to allow the correct access. If you need credentials/have difficulty connecting, contact [mailto:software@shadowrobot.com]
+
+As the example we're constructing is for Hand E, these options are not required as the image is public.
+
+### Naming your image: ```-n``` and ```-sn```
+
+The name that you give your container should be descriptive of the task for which you wish to it. This will make it easier to keep your containers organised and also to select the correct one if you have multiple on a machine. The container name, specified by the ```-n``` flag should be unique, as **it will be overwritten if you run the one-liner with the same value again**.
+
+The ```-sn``` flag gives the name of the desktop link that will be created for starting the driver. Keeping the container name and shortcut name the same will reduce the chance of confusion.
+
+For example. let's say you want a docker to test a kinetic Hand E, you could use the name "hand_e_kinetic":
+
+```bash <(curl -Ls http://bit.do/launch-sh) -r true -i shadowrobot/dexterous-hand:kinetic -n hand_e_kinetic -sn hand_e_kinetic```
+
+### Setting port and config branch: ```-e``` and ```-b```
+#### Ethernet port
+To select the correct ethernet port when starting a docker, use the one-liner option  ```-e ETH_PORT``` where *ETH_PORT* is the name of the port to which the robot is connected. If you don't know which port to use, typing ```sudo eepromtool``` in any running docker will give you the name of any port with Shadow hardware connected:
```

Comment:
Maybe here add command line for creation of Docker container without Ethernet port and running in Terminator mode and after that command with Ethernet port should have the same container name to destroy unneeded temporary one.
Just for completeness in case if people do not have any Docker container on their machine.